Koha/Biblio/Availability/Hold.pm

```diff
@@ -209,7 +210,6 @@ sub _item_looper {
             push @{$self->{'item_availabilities'}}, $item_availability;
             $count++;
         } else {
-            my $unavails = $item_availability->unavailabilities;
             # If Item level hold is not allowed and it is the only unavailability
             # reason, push the item to item_availabilities.
             if ($item_availability->unavailable == 1 && exists
```

Comment:
After the change, this first if-block becomes dead code. We should just drop it.

Reply:
Thanks. This code is dropped now.